src/master/concepts.md

```diff
@@ -1,109 +1,50 @@
 # Concepts
 
-Download the [immudb short research paper](https://codenotary.com/technologies/immudb/) to learn about the technical foundations of immudb.
+Download the [immudb short research paper](https://codenotary.s3.amazonaws.com/Research-Paper-immudb-CodeNotary_v3.0.pdf) to learn about the technical foundations of immudb.
 
 <WrappedSection>
 
-## Adding data
+## Consistency and state signature
 
-This section is not yet ready for immudb 0.9. We are working on it in order to improve it and we are close to deliver. Stay tuned!
+Immudb consistency can be verified by any external client or auditor by calculating transaction state and comparing it with one returned from immudb. The state is represented by the root digest of a Merkle tree and is calculated for every database transaction. It allows for verification at each transaction.
```

Comment:
The state is represented by a digest, which is calculated as part of the transaction commit phase. Calculating such a value in an accumulative manner ensures that any change already made cannot be reversed. 

-> because the state is not matching the root hash of the hash tree

Reply:
👍